src/components/pages/donation_form/DonorSummarySection.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<div aria-live="polite">
+		<div>
```

Comment:
Get rid of these 2 divs and only include the content. It's important that the html of the summary content matches with the html in the pattern https://spenden.wikimedia.de/pattern-library/summary

Reply:
The `aria-live="polite"` attribute should be added to the summary component: `<div class="summary switcher" aria-live="polite">` You could also update the `markup.html` in the summary pattern